app/hooks/useVerifyCredential.ts

```diff
@@ -1,13 +1,14 @@
 import { useState, useCallback } from 'react';
 import { ResultLog, verifyCredential } from '../lib/validate';
 import { CredentialError } from '../types/credential';
-import { Cache, CacheKey } from '../lib/cache';
 import { CredentialRecordRaw } from '../model';
 import { useFocusEffect } from '@react-navigation/native';
+import { LruCache } from '@digitalcredentials/lru-memoize';
 
 /* Verification expiration = 30 days */
 const VERIFICATION_EXPIRATION = 1000 * 60 * 60 * 24 * 30;
```

Comment:
```suggestion
const VERIFICATION_EXPIRATION = 1000 * 30;
```
I suggest we make it closer to 30 seconds, than 30 days.

Reply:
@dmitrizagidulin I thought we wanted to keep checking once a month because it causes the app to be very slow otherwise?

Reply:
Definitely not once a month. If a VC is revoked or expires, it needs to update on the wallet home view eventually (so like, within a minute or so).